web/src/app/lists/PaginatedList.js

```diff
@@ -170,27 +184,38 @@ export class PaginatedList extends React.PureComponent {
 
     // provide a message to display if there are no results
     emptyMessage: p.string,
+
+    // renders as an infinite list instead of the default
```

Comment:
Maybe we can describe this in terms of specific behavior. E.g. "if set, `loadMore` will be called when the user scrolls towards the bottom of the list"

Reply:
Works for me